src/components/buttons/ButtonStack.tsx

```diff
@@ -25,11 +25,7 @@ interface ButtonStackProps extends ViewProps {
 }
 
 const ButtonStack = ({ children, style }: ButtonStackProps) => (
-  <View style={style}>
-    {children?.map((child, i) => (
-      <ButtonContainer key={i}>{child}</ButtonContainer>
-    ))}
-  </View>
+  <View style={style}>{children?.map((child, i) => <ButtonContainer key={i}>{child}</ButtonContainer>)}</View>
```

Comment:
Linting was broken for some reason so after upgrading the eslint related packages, this changes was required (also for next file)